advtraj/utils/grid.py

```diff
@@ -90,7 +90,13 @@ def wrap_periodic_grid_coords(
         else:
             x_max += dx
 
-        ds_posn_copy[c].values = wrap_posn(
-            ds_posn_copy[c].values, x_min=x_min, x_max=x_max
-        )
+        wrapped_x = wrap_posn(ds_posn_copy[c].values, x_min=x_min, x_max=x_max)
+        d = ds_posn_copy[c].dims
```

Comment:
I think we need a comment here about what's going on. Doesn't quite make sense to me from the code directly

Reply:
Done.